x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_explorer/analytics_collection_explorer_table.tsx

```diff
@@ -292,6 +294,8 @@ export const AnalyticsCollectionExplorerTable = () => {
   const handleTableChange = ({ sort, page }: Criteria<ExploreTableItem>) => {
     onTableChange({ page, sort });
   };
+  const startNumberItemsOnPage = pageSize * pageIndex + Number(!!items.length);
```

Comment:
`Number(!!items.length)` is a little too clever for my taste. if you're open to it i'd like something more explicit. a ternary operator perhaps? or maybe just a comment explaining what this is and why it's here?

Reply:
@sloanelybutsurely Could you look on updated pr?